Resources/NetHook2/GenerateVersionInfo.ps1

```diff
@@ -0,0 +1,34 @@
+﻿$ErrorActionPreference = 'stop'
+
+Function Test-GitAvailable {
+    Try {
+        If (Get-Command git) {
+            return $true
+        }
+    }
+    Catch {
+        return $false
+    }
+}
+
+$buildDate = Get-Date -Format "yyyy-MM-dd HH:mm:ss K"
+
+if (Test-GitAvailable) {
```

Comment:
Don't you want to add default/fallback variables if it happens to be not available?

Reply:
the strings should become empty string, I need to test the bool.